ooniapi/services/oonifindings/tests/conftest.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+import pytest
+
+import time
+import jwt
+
+from fastapi.testclient import TestClient
+from clickhouse_driver import Client as Clickhouse
+
+from oonifindings.common.config import Settings
+from oonifindings.common.dependencies import get_settings
+from oonifindings.main import app
+
+
+def run_migration(path: Path, click: Clickhouse):
+    sql_no_comment = "\n".join(
+        filter(lambda x: not x.startswith("--"), path.read_text().split("\n"))
+    )
+    queries = sql_no_comment.split(";")
+    for q in queries:
+        q = q.strip()
+        if not q:
+            continue
+        click.execute(q)
+
+
+@pytest.fixture
+def clickhouse_migration(clickhouse):
```

Comment:
We should for the time being swap this out for pytest-docker based approach:  https://github.com/ooni/data/blob/main/oonipipeline/tests/conftest.py#L39